### DISCUSSION TITLE
Changing slot date format

### DISCUSSION THREAD

**Discussion by zasorindesu** [eouia/MMM-CalendarExt2#276] [Q&A] [answered]
Hey! I've looked through the documentation multiple times and haven't figured out how to disable month and date in the slot header, daily view. I would like to have only "today", "tomorrow", "later", etc. At the moment it says "Today XX of February" in the header. Any help is highly appreciated!
![msg91253297-206849](https://user-images.githubusercontent.com/126142563/220839249-9b8f41d0-4a5a-48ae-8f48-f25d883dbb33.jpg)

**Comment by klaernie** [comment, marked as answer]
Did you take a look at `dateFormat` and `timeFormat`?
https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/blob/main/docs/Event-Time.md

**Reply by zasorindesu** [reply]
Yep, I've looked at these instructions and tried to play with them. As I understood they are responsible for the time and date formats living within the event.

**Reply by klaernie** [reply]
Oh, missed that detail - you need to set the right option depending on the view type, as that's a property of the view.
https://github.com/MMM-CalendarExt2/MMM-CalendarExt2/blob/main/docs/Configuration/Views/daily.md (for the daily view it's easy to see)

**Reply by zasorindesu** [reply]
You were correct! What I was looking for  was `slotSubTitleFormat: 'D/M/YYYY'`. Works like a charm! Thank you!
